modules/ROOT/pages/functions/graph.adoc

```diff
@@ -89,6 +89,22 @@ Properties for all graphs on the current composite database are returned.
 
 |===
 
+.Query
+[source, cypher, indent=0]
+----
+UNWIND graph.names() AS name
+WITH name, graph.propertiesByName(name) AS props
+WHERE any(tag IN props.tags WHERE tag IN ["A"])
+CALL {
+  USE graph.byName(name)
+  MATCH (n)
+  RETURN n
+}
+RETURN n
+----
+
+Returns all nodes from a subset of graphs that have a tags property containing "A".
```

Comment:
```suggestion
Returns all nodes from a subset of graphs that have a `tags` property containing `"A"`.
```

Reply:
@smietana, into which version should this PR go?

Reply:
Function graph.propertiesByName was added in 5.2, so if we have minor version granularity it should go to 5.2 and above otherwise 5.x.